app/__init__.py

```diff
@@ -3,21 +3,56 @@
 Copyright (c) 2020 - DevsBranch
 """
 
+import os
 from flask import Flask
+from celery import Celery
 from flask_marshmallow import Marshmallow
 from flask_login import LoginManager
 from flask_sqlalchemy import SQLAlchemy
 from importlib import import_module
 from flask_migrate import Migrate
 from flask_jwt_extended import JWTManager
 from decouple import config as db_config
+from config import config_dict
+
+CELERY_TASK_LIST = [
+    "app.celery_utils",
+]
+
+
+def make_celery(app):
+    celery = Celery(app.import_name,
+                    include=CELERY_TASK_LIST)
+    celery.conf.update(app.config)
+    celery.conf.update(
+        broker_url=os.environ.get("REDIS_URL") or "redis://localhost:6379/0",
```

Comment:
I think for broker_url this  would be enough `"redis://localhost:6379"` 
unless  it does not work

Reply:
I have used "REDIS_URL" because Heroku uses it as an environmental variable for the Redis server. So there is no need to change the Redis server URL in the app every time when deploying the app to Heroku